projects/assets/src/translations/en/common.ts

```diff
@@ -48,6 +48,15 @@ export const common = {
   },
   searchBox: {
     placeholder: 'Search here...',
+    ariaLabelInput: 'Search here...',
+    ariaLabelSuggestions: 'typing suggestions',
+    ariaLabelProducts: 'product results',
+
+    initialDescription:
+      'When autocomplete results are available use up and down arrows to review and enter to select. Touch device users, explore by touch or with swipe gestures.',
+    suggestionsAnd: ' suggestions and ',
+    productsAreAvailable: ' products are available.',
```

Comment:
```suggestion
	result: '{{ suggestionsCount }} suggestions and {{ productsCount }} products are available.'
```

Reply:
We have 2 options:
1. split it to 2 pluralizable sub-sentences
2. leverage low level API of i18next https://www.i18next.com/translation-function/nesting#passing-options-to-nestings

I prefer the option 1. It's because some day we might want to change the i18n engine under the heed (i.e. to the native Angular one, when it fits our needs). And then it will be much easier, if only we avoid using super power low level API of i18next.

Reply:
fixed